05week/checkers.js

```diff
@@ -27,6 +27,29 @@ class Board {
       }
     }
   }
```

Comment:
I am missing alot of the functionality of this assignment. Did you check all of it in?